### PR TITLE
Add support for trace log level

### PR DESCRIPTION
**Pull request by kraih**
This PR adds `log_trace` and moves a few not particularly useful (in day to day debugging) but very verbose log messages to the trace level.
```
[2021-10-21T10:00:12.977339+02:00] [debug] Received JobToken: A1234567890
[2021-10-21T10:00:12.977642+02:00] [debug] API key from client: *B1234567890*
[2021-10-21T10:00:12.978024+02:00] [debug] API key from client: *C1234567890*
[2021-10-21T10:00:12.978031+02:00] [debug] API key from client: *D1234567890*
```

Progress: https://progress.opensuse.org/issues/101379

### CODE REVIEW

File: dependencies.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ common_requires:
   perl(Filesys::Df):
   perl(Getopt::Long):
   perl(Minion): '>= 10.22'
-  perl(Mojolicious): '>= 9.11'
+  perl(Mojolicious): '>= 9.20'
```
Comment (kalikiana) [inline dependencies.yaml:42]: Please mention that in the description

**Review by kalikiana — APPROVED** [review]
Looks pretty straightforward to me, including the obvious test addtions.

Please make it clear that this requires Mojo 9.20, though. I don't know if there is any issue with that, but it'll be important for anyone who hasn't upgrade yet.

**Review by Martchus — APPROVED** [review]

**Comment by kalikiana** [comment]
Well, seems like mergify really doesn't care for open comments anymore...

**Comment by okurz** [comment]
> Well, seems like mergify really doesn't care for open comments anymore...

It never did. mergify waits for open requested reviews as well as requested changes but you approved and Martchus approved, that's it

**Pull request merged**